modules/core/config/MobileConfig.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+/**
+ * Model for mobile backend configuration
+ */
+public struct MobileConfig: Codable {
+    // Service namespace
+    public var namespace: String
```

Comment:
Every prop in model should be optional, because sometimes backend can send you nil/null value